cohortextractor/presto_utils.py

```diff
@@ -1,8 +1,10 @@
+import readline  # noqa
```

Comment:
I guess something magical happens when you import this? A comment would be helpful

Reply:
Yes -- it adds readline behaviour to `input()`.